tuned/profiles/functions/function_ternary.py

```diff
@@ -0,0 +1,24 @@
+import re
+from . import base
+
+class ternary(base.Function):
```

Comment:
Personally I'd reserve the name 'ternary' for an actual ternary operator, i.e. `x ? foo : bar`. The name should make it clear that the function has the semantics of a ternary operator, but that the condition is a regex match. I'd name it `regex_search_ternary`. (Also note the change match -> search. This better reflects the python implementation.)